### DISCUSSION TITLE
Automatically use image's EXIF meta data in Asset fields

### DISCUSSION THREAD

**Discussion by nitech** [craftcms/cms#8699] [Ideas]
When an image containing EXIF meta title and description is uploaded, it would be nice that these texts automatically were put inside the appropriate field on the asset. 

As far as I know, the assets only have one default field: title. So remaining fields would have to be mapped after creating each respective field.

**Comment by petenice** [comment]
Is this possible?

**Reply by brandonkelly** [reply]
We fetch the image orientation from EXIF data, but nothing else currently.

**Reply by petenice** [reply]
Thanks, client is tagging photos in Lightroom and wondering if tags would be available after after import to Assets. Sounds like nope. Much appreciated!
